platform/darwin/src/NSCoder+MGLAdditions.h

```diff
@@ -0,0 +1,10 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+@interface NSCoder (MGLAdditions)
+
+- (void)encodeMGLCoordinate:(CLLocationCoordinate2D)coordinate forKey:(NSString *)key;
```

Comment:
Publicly vending these methods would’ve made more sense if we could come up with a way to encode and decode the coordinate as a struct. But with the dictionary-based implementation is dependent on the keys we choose, so unfortunately it isn’t portable enough to expose publicly. MGLCode+MGLAdditions_Private.h can be folded into this header, which can be marked with Project visibility. Sorry for the detour.